src/components/shared/errors/PageError.tsx

```diff
@@ -32,7 +32,11 @@ const PageError = ({ graphic = 'dad', text, cause, details, action, noReload = f
     return (
         <div className="flex max-w-3xl gap-4 max-[960px]:flex-col mb-16" role="status" aria-live="polite">
             <div className="relative h-64 w-96 grow self-center">
-                {graphic === 'dad' ? <Image src={pageErrorDad} alt="" fill /> : <Image src={notFoundMom} alt="" fill />}
+                {graphic === 'dad' ? (
+                    <Image src={pageErrorDad} alt="Mann som sitter i godstol og leser en avis" fill />
+                ) : (
+                    <Image src={notFoundMom} alt="Dame som ser på 404-tall som ligger på bakken" fill />
+                )}
```

Comment:
Tror ikke denne alt-teksten faktisk gir noe nytteverdi for brukere med skjermleser. Det er en ren visuell greie som ikke gir noe informasjon.